sdk/core/azure-core/src/main/java/com/azure/core/implementation/UnexpectedLengthException.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.storage.blob;
+package com.azure.core.implementation;
```

Comment:
This should go in the implementation.exception package

Reply:
Done.